src/main/scala/authentikat/jwt/JsonWebSignature.scala

```diff
@@ -52,7 +52,7 @@ object JsonWebSignature {
 
   abstract class Algorithm
 
-  case object none extends Algorithm
+  case object NONE extends Algorithm
```

Comment:
Yes it think it can be `None`, but for the reader that might be confusing because it looks like an `Option`'s `None`. That was the reason for me not to use the Scala naming convention.